drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -257,6 +257,10 @@ def load_definition(form_definition:, identifier:, role:, title: nil)
       record.title = title if title.present?
       record.status = Hmis::Form::Definition::PUBLISHED
 
+      # Ensure HUD rules are set
+      changed_link_ids = record.set_hud_requirements # return chagned link ids
+      raise "hud rules wrong for #{identifier}: #{changed_link_ids}" if changed_link_ids.any?
+
```

Comment:
This is a check to ensure I didn't break anything. Once comfortable with this I can remove all the `"rule"` settings from the JSON